ChangeLog.d/issue1792.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fix a bug in ECDSA that would cause it to fail when the payload is all-bits
```

Comment:
The PR description says ECJPAKE, the changelog entry says ECDSA, I believe the library changes fix both, the tests have been changed only for ECJPAKE. Please update the PR title, the changelog entry and the tests.

Reply:
I believe the  ChangeLog entry is correct. This addresses #1792 which is all about ECDSA. The ECDSA fix introduced an issue in EC J-PAKE which was then fixed in the same PR. Since there was no problem with EC J-PAKE in any released version, I don't see a need for a ChangeLog entry about it. (The changes in the tests are about changes in error cases where documentation was sufficiently vague that I don't think it counts as a behaviour change.)

Reply:
Though I think "payload" is confusing and we should say "hash" instead.